.release-it.json

```diff
@@ -0,0 +1,8 @@
+{
+  "git": {
+    "commitMessage": "chore: release v${version}"
```

Comment:
If it makes a commit on release, we should also push that commit. But I don't see support for that in the current CI setup.

Reply:
We  can probably give the CI push access. See https://docs.github.com/en/actions/using-jobs/assigning-permissions-to-jobs

Reply:
But if we enable branch protection it won't be possible anymore. So we either need to take the approach of not commiting when we make a release (as we do in more libraries, by not looking at package.json for the next version), or we need to disable brancn protectino and allow pushing to main, or we need to make a PR first to increase the version, and based on that trigger a release.

I'm fine with a simple solution for now

Reply:
Good catch!